src/platforms/apple/common/sourcecontext.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Use Source context
+sidebar_order: 2
```

Comment:
When you check preview, is this showing up where you expect in the sidebar? I thought it would be right under "Uploading Debug Symbols" but it's not.

Reply:
I see it below `Uploading Debug Symbols` https://sentry-docs-git-feat-source-context.sentry.dev/platforms/apple/guides/ios

Reply:
It was updated